sda-commons-server-cloudevents/asyncapi.yaml

```diff
@@ -251,7 +253,9 @@ components:
           type: "string"
           default: "PARTNER_CREATED"
           examples:
-          - "CONSENT_VERSION_CREATED"
+          - "com.sdase.relocation.service.customer.moved"
+          - "com.sdase.consent.service.consent.agreed"
+          - "com.sdase.offer.service.contract.offered"
           minLength: 1
           description: "`PARTNER_CREATED`"
```

Comment:
Should also match the examples.

Reply:
Good catch. [fixed](https://github.com/SDA-SE/sda-dropwizard-commons/pull/1783/commits/a5123945bf6e92526a12ff4874efb41739d68bfb)